### ISSUE TITLE
mac OS Ventura: VirtualHIDDevice driver "allow" button never shows up

### ISSUE THREAD

**Issue by Mebius973** [pqrs-org/Karabiner-Elements#3294]
Hello,

I'm trying to install KE on a M1 pro mbp upgraded to Ventura. SIP is disabled and System Security Policy is set to allow users to load kernel extensions.

I don't get the "allow" button to show up to grant permissions for the driver. I've tried the procedure on the popup. I've tried uninstalling/reinstalling KE. I've tried launching "Karabiner-DriverKit-VirtualHIDDeviceClient"  manually to see what's going on ("virtual_hid_device_service_server::create_rootonly_directory permissions error: Operation not permitted").

Can you help me on this please?

OS Version: Ventura 13.0.1
KE version:  14.10.0

**Comment by ksuchoi216** [comment]
I faced the same problem about "the Karabiner-DriverKit-VirtualHIDDeviceClient" popup message. I reinstalled the karabiner several times, and also I did deactivation and reactivation for allowing the driverki.
However, it is still happening again and again. I shared my log as follows:

"[grabber] virtual_hid_keyboard is not ready. Please wait for a while"

OS Version: Ventura 13.0.1
KE version: 14.10.0
CPU: Intel Iris Plus Graphics 1536 MB

**Comment by Mebius973** [comment]
@ksuchoi216 FYI, I personnaly used karabiner to swap keys on keyboards which where improperly detected by macos. If your need is the same, here is the solution I found to achieve this without KE: https://github.com/Mebius973/KeyRemapping

**Comment by myst3k** [comment]
Having the same issue on a new laptop. Tried 14.10.0, and Beta 14.10.1. I have also put the system into low security mode, still cant get the button to show up.

**Comment by Swivelgames** [comment]
Came back from a bit of a vacation and the same is happening to me. Except it happened after installing a security update for macOS Monterey (I'm on Monterey 12.6.1)

**Comment by LeonardoRick** [comment]
Having the same problem on Ventura

**Comment by Swivelgames** [comment]
Indeed. Still appears to be an issue. Complains every time I start up my laptop lol
